### PR TITLE
Implemented wayland display resize channel.

### PR DESCRIPTION
**Pull request by akallabeth**
* Implements the display resize for wayland
* Currently only single monitor
* Requires #5202 to actually display the changed resolution correctly

### CODE REVIEW

File: client/Wayland/wlf_disp.c [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+
+	ret = IFCALLRESULT(CHANNEL_RC_OK, disp->SendMonitorLayout, disp, nmonitors, layouts);
+	free(layouts);
+	return ret;
+}
+
+BOOL wlf_disp_handle_configure(wlfDispContext* disp, int32_t width, int32_t height)
+{
+	if (!disp)
+		return FALSE;
```
Comment (hardening) [inline client/Wayland/wlf_disp.c]: duplicated check

Comment (akallabeth) [reply client/Wayland/wlf_disp.c]: exported in `wlf_disp.h`, no internal static

Comment (hardening) [reply client/Wayland/wlf_disp.c]: @akallabeth well I don't like the check (as it's not really exported as just used internally, it's present in the header file just for convenience). My point was more that you have exactly the same check 2 lines below.

Comment (akallabeth) [reply client/Wayland/wlf_disp.c]: @hardening ok, that duplication was not intentional

**Pull request merged**